app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,29 @@
+class PostsController < ApplicationController
+  require "will_paginate/array"
+  PAGE_SIZE = 4
+
+  def index
+    page = (params[:page] || '1').to_i
+
+    client = Tumblr::Client.new({
```

Comment:
That's a good idea. Using Tumblr as our blog engine has turned out to be kind of a mess in general though. I'm thinking we may be better served by migrating the blog to WordPress, possibly along with all the other static content, and using Rails only for the actual app. That way the non-technical people can update stuff without bothering devs. Thoughts?